### PR TITLE
Added TrustServerCertificate param in connectionInfo

### PR DESCRIPTION
**Pull request by himanshu634**
Added TrustServerCertificate param in connectionInfo as described [here](https://github.com/Canner/WrenAI/issues/698).

### CODE REVIEW

**Comment by himanshu634** [comment]
@wwwy3y3 @grieve54706  It is ready to review !

**Review by grieve54706 — CHANGES_REQUESTED** [review]
It should not be hard-coded. It should be an option on the website that lets users decide whether their database needs this.

**Comment by himanshu634** [comment]
I already doubted it. Anyways, I will do it.

**Comment by himanshu634** [comment]
@grieve54706 Please review it!, I have added Enable Trust Server Certificate user side as well.

**Comment by wwwy3y3** [comment]
@andreashimin will help review

File: wren-ui/src/apollo/server/dataSource.ts [hunk 1]
```diff
@@ -123,9 +123,18 @@ const dataSource = {
...
-      return { host, port, database, user, password };
+      return {
+        host,
+        port,
+        database,
+        user,
+        password,
+        kwargs: {
+          TrustServerCertificate: trustServerCertificate,
+        },
```
Comment (andreashimin) [inline wren-ui/src/apollo/server/dataSource.ts:136]: According to #698, you need to convert it to string `YES`
if false then don't provide the property

File: wren-ui/src/components/pages/setup/dataSources/SQLServerProperties.tsx [hunk 2]
```diff
@@ -88,6 +88,13 @@ export default function SQLServerProperties(props: Props) {
       >
         <Input placeholder="SQL Server database name" disabled={isEditMode} />
       </Form.Item>
+      <Form.Item
+        label="Enable Trust Server Certificate"
+        name="trustServerCertificate"
+        valuePropName="checked"
+      >
+        <Switch />
+      </Form.Item>
```
Comment (andreashimin) [inline wren-ui/src/components/pages/setup/dataSources/SQLServerProperties.tsx:97]: Please help make adjustment to the property
- Default to checked
- Add `extra` prop to `Form.Item` with value: `This parameter is used to skip server certificate validation. If you are using a trusted certificate, you can disable it.`

**Review by andreashimin — COMMENTED** [review]
Hi @himanshu634, I saw there is a conflict in this PR,
Could you help rebase it? Thank you

**Comment by himanshu634** [comment]
@andreashimin Please check now, thanks.

**Review by andreashimin — APPROVED** [review]
LGTM

**Comment by andreashimin** [comment]
Hi @himanshu634 ,

Thank you for your contribution 👍🏼 
You can now DM @chilijung on Discord with https://github.com/Canner/WrenAI/issues/698 to get the swag form link.

**Pull request merged**